ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -167,30 +170,12 @@ public void handleGet(RestRequest restRequest, RestResponseChannel restResponseC
 
   @Override
   public void handlePost(RestRequest restRequest, RestResponseChannel restResponseChannel) {
-    long processingStartTime = System.currentTimeMillis();
-    long preProcessingTime = 0;
     handlePrechecks(restRequest, restResponseChannel);
```

Comment:
The pre check actually checks whether either of `restRequest` or `restResponseChannel` is null. If that is so, then there is no point proceeding. This essentially means there is a programming error. So it is still required.